docs/modules/ROOT/pages/how-to-guides/managing-environments/proc_creating_your_own_environment.adoc

```diff
@@ -1,31 +1,63 @@
+You can configure {ProductName} to deploy applications to your own cloud-hosted cluster, outside the default development environment. By using this option to bring your own cluster (BYOC), you can deploy your applications to an environment that meets your specific needs. And if you bring your own cluster, you can still access all the features and benefits of {ProductName}. 
 
 .Prerequisites
 
-* You must have access to credentials with full read and write access to the namespace that you provided to {ProductName}.   
-* When you upload a `kubeconfig` file, it must include a credential with full read and write access to your provided namespace. 
-* The supplied `kubeconfig` file must use token-based authentication to connect to your cluster. Other mechanisms, such as certificate-based authentication, are not yet supported. 
-* The context in your `kubeconfig` file must point to the cluster you want to use at the time you upload the file. 
-
-.Procedures
-
-To create your own environment and upload your own `kubeconfig` file, complete the following steps in the {ProductName} UI: 
-
-. Go to *Environments* > *Create Environment*. 
-. Complete all the required fields for *Create Environment*.  
+* You must have credentials for full read and write access to the namespace for your cluster.   
+
+.Procedure
+
+To create your own environment, you first need to create a functional `kubeconfig` file. Then, using that file, you add the cluster as a new environment in the {ProductName} UI.
+
+. In your preferred text editor, create a `kubeconfig` file based on the following example:
+
++
+
+[source]
+--
+apiVersion: v1
+clusters:
+- cluster:
+    server: https://api.sandbox-m2.ll9k.p1.openshiftapps.com:6443
+  name: api-sandbox-m2-ll9k-p1-openshiftapps-com:6443
+contexts:
+- context:
+    cluster: api-sandbox-m2-ll9k-p1-openshiftapps-com:6443
+    namespace: rhn-support-csears-dev
+    user: rhn-support-csears/api-sandbox-m2-ll9k-p1-openshiftapps-com:6443
+  name: rhn-support-csears-dev/api-sandbox-m2-ll9k-p1-openshiftapps-com:6443/rhn-support-csears
+current-context: rhn-support-csears-dev/api-sandbox-m2-ll9k-p1-openshiftapps-com:6443/rhn-support-csears
+kind: Config
+preferences: {}
+users:
+- name: rhn-support-csears/api-sandbox-m2-ll9k-p1-openshiftapps-com:6443
+  user:
+    token: sha256~<remainder of token omitted>
+--
+
++
+
+. In the {ProductName} UI, go to *Environments* > *Create Environment*. 
+. Enter a name for the new environment.  
 . Complete all the required fields for *Cluster information*: 
 .. For *Select cluster*, select that you are bringing your own cluster.  
 .. For *Cluster type*, select *Non-OpenShift* or *OpenShift*. 
-. Upload your `kubeconfig` file.
-. Attach the namespace to the cluster that you selected for this application.
+.. Upload your `kubeconfig` file, or paste its contents into the space provided.
+.. Specify the namespace for the cluster that you are using.
```

Comment:
Choose to ignore

Here we have specified the kind of namespace. I believe adding annotations in the example `Kubeconfig` file will add clarity :)